packages/app/src/specs/SpecsList.vue

```diff
@@ -175,6 +180,19 @@ const { containerProps, list, wrapperProps, scrollTo } = useVirtualList(treeSpec
 // If you are scrolled down the virtual list and list changes,
 // reset scroll position to top of list
 watch(() => treeSpecList.value, () => scrollTo(0))
+
+function handleCtrlClick () {
+  // noop intended to reduce the chances of opening tests multiple tabs
+  // which is not a supported state in Cypress
+}
+
+function getIdIfDirectory (row) {
+  if (row.data.isLeaf && row.data) {
+    return null
+  }
+
+  return row.data.data.relative.replace(row.data.data.baseName, '')
```

Comment:
Is it worth prefixing this id value to better guarantee uniqueness?

```suggestion
  return `speclist-${row.data.data.relative.replace(row.data.data.baseName, '')}`
```